cluster/restore-from-backup.sh

```diff
@@ -153,16 +153,15 @@ if [ "${ETCD_API}" == "etcd2" ]; then
   mkdir -p "${BACKUP_DIR}/member/snap"
   mkdir -p "${BACKUP_DIR}/member/wal"
   # If the cluster is relatively new, there can be no .snap file.
-  mv *.snap "${BACKUP_DIR}/member/snap/" || true
-  mv *.wal "${BACKUP_DIR}/member/wal/"
+  mv ./*.snap "${BACKUP_DIR}/member/snap/" || true
+  mv ./*.wal "${BACKUP_DIR}/member/wal/"
 
   # TODO(jsz): This won't work with HA setups (e.g. do we need to set --name flag)?
   echo "Starting etcd ${ETCD_VERSION} to restore data"
-  image=$(docker run -d -v ${BACKUP_DIR}:/var/etcd/data \
+  if ! image=$(docker run -d -v ${BACKUP_DIR}:/var/etcd/data \
```

Comment:
functionally equivalent, but avoids the pitfalls of trying to check `$?` and potentially having it overwritten by something before you get to reading it.